rllib/evaluation/rollout_worker.py

```diff
@@ -903,6 +909,17 @@ def sample(self) -> SampleBatchType:
             >>> print(worker.sample()) # doctest: +SKIP
             SampleBatch({"obs": [...], "action": [...], ...})
         """
+
+        t_offset = self.config._kill_rollout_workers_t_offset
+        uptime = time.time() - self._creation_time
```

Comment:
this decays too fast though. we need to change it into hours.
also assert_healthy is only called during ``__init__`` time, so we should probably move this into ``sample()``

Reply:
You misread this. It is in sample():. Also what do mean it decays fast? It is fully controlled with those hps. You can make scale arbitrarily large so that it doesn't decay fast.